README.md

```diff
@@ -62,10 +62,10 @@ Check out [plugins documentation](./docs/plugins.md).
 First, clone the repo via git:
 
 ```bash
-git clone https://github.com/KELiON/cerebro.git cerebro
+$ git clone https://github.com/KELiON/cerebro.git cerebro
```

Comment:
Yes, that's right, it's a sign of the dollar needed to show that this is a team that needs to be executed in the terminal. Now the dollar sign is added for other commands, so it's logical to add it to `git clone`.